src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -432,6 +433,7 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
 
     using namespace ngraph::pass::low_precision;
     if (useLpt) {
+        CPU_LPT_SCOPE(LowPrecisionTransformations);
```

Comment:
we have several `if (useLpt)` in this file, should we use this `CPU_LPT_SCOPE` in all such places?

Reply:
Fixed in cbaf98dc955a0dcbf9a2bc649a271fab97eb192e. Thank you!